internals/maintenance_scripts.py

```diff
@@ -500,3 +505,74 @@ def get_template_data(self, **kwargs):
     ndb.put_multi(batch)
 
     return f'{count} Feature entities updated of {len(features_by_id)} available features.'
+
+
+class CreateOriginTrials(FlaskHandler):
+
+  def handle_creation(self, stage: Stage, stage_dict: StageDict) -> str | None:
+    """Send a flagged creation request for creation."""
```

Comment:
This description seems circular.

Reply:
Yes, it's not very clear. Updated it to be more specific. 🙂